### PR TITLE
CPDP-2262: send email in chosen language

### PR DESCRIPTION
**Pull request by mikail-khan**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -276,4 +276,6 @@ s3 {
     file-download-timeout = 2 minutes
     upstream-element-limit-scale-factor = 200 # do not adjust
     max-file-download-size-in-mb = 5
-}
\ No newline at end of file
+}
+
+httpHeadersWhitelist=["Accept-Language"]
```
Comment (AndyHWChung) [inline conf/application.conf:281]: we're supposed to use the term `allowList` now instead of `whitelist` and `denyList` instead of `backList`

Comment (mikail-khan) [reply conf/application.conf:281]: you can't - this is the key used in `http-verbs` and that can't change unless Plat Ops release a new version of the library

Comment (AndyHWChung) [reply conf/application.conf:281]: or is this even something we have control over?

Comment (AndyHWChung) [reply conf/application.conf:281]: would we not need `+=` if the config param comes from another libary?

Comment (mikail-khan) [reply conf/application.conf:281]: we don't have control over this. 
we don't need the += - this is a list specific to the service using the http-verbs library

Comment (AndyHWChung) [reply conf/application.conf:281]: :+1: didn't see your reply before writing my other comments

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**